src/frontend/packages/lib_common/package.json

```diff
@@ -37,6 +37,7 @@
     "babel-loader": "8.2.5",
     "babel-plugin-react-intl": "8.2.25",
     "babel-preset-react": "7.0.0-beta.3",
+    "cross-env": "7.0.3",
```

Comment:
You can add this dependency in all packages and apps because we are using `TZ=UTC` in all of them.

Reply:
And no `yarn.lock` is modified. You must add a dependency using `yarn add`, not by editing yourself the `package.json` file. Otherwise the yarn lock is not modified, nor commited.